core/sr-io/without_std.rs

```diff
@@ -21,18 +21,41 @@ pub use rstd;
 pub use rstd::{mem, slice};
 
 use core::intrinsics;
-use rstd::vec::Vec;
+use rstd::{vec::Vec, ops::Deref};
 use hash_db::Hasher;
 use primitives::Blake2Hasher;
 
+use cfg_if::cfg_if;
+
+cfg_if! {
+	if #[cfg(feature = "wasm-nice-panic-message")] {
+		fn wasm_nice_panic_message() -> bool {
```

Comment:
Couldn't this whole `cfg_if` be replaced with

```rust
fn wasm_nice_panic_message() -> bool { cfg!(feature = "wasm-nice-panic-message") }
```

Reply:
I don't wanted any implementation to leak into the wasm files, if we not request it. So, we don't grow the wasm file.

Reply:
I don't understand how that connects to this block of code. `cfg_if` seems to do the exact same.

Reply:
If `wasm-nice-panic-message` is enabled, it will generate code that prints the full panic message on the host and in that implementation the result is not always `false`. The result just tells us if we fallback to the default implementation of just printing the line + file.

Reply:
Nevertheless, I already changed it locally to a different implementation.